release/nightly_tests/dataset/read_and_consume_benchmark.py

```diff
@@ -35,15 +35,19 @@ def parse_args() -> argparse.Namespace:
 
 
 def main(args):
-    benchmark = Benchmark("read-and-consume")
-    read_fn = get_read_fn(args)
-    consume_fn = get_consume_fn(args)
+    benchmark = Benchmark()
 
     def benchmark_fn():
+        read_fn = get_read_fn(args)
+        consume_fn = get_consume_fn(args)
```

Comment:
Drive-by change to move defining the pipeline into the benchmark. On open-source Ray, calling but not executing functions like `read_parquet` can be slow